docs/examples.rst

```diff
@@ -195,6 +195,54 @@ the single dot notation:
     $ tmt run test --name .
 
 
+Create Tests
+------------------------------------------------------------------
+
+Use ``tmt tests create`` to create a new test. You need to specify
+the template using the ``--template`` option. You can also specify the test
+metadata template using ``--metadata-template``. If only ``--template``
+is specified, the same template name is used for the metadata template.
+
+There is an option to create custom templates. To do so, place your test
+template files inside the directory ``~/.config/tmt/templates/test`` and
+your test metadata files inside the ``~/.config/tmt/templates/test_metadata``.
+
+Templates must have the ``.j2`` file extension.
```

Comment:
So this means the templates are Jinja2 based, Could we provide more information about the supported filters, etc? I guess we currently support only the default ones?

Reply:
Also I would appreciate a link to the directory of current tmt provided templates, to take some inspiration from maybe?

Reply:
WRT filters: there's more, https://github.com/teemtee/tmt/blob/main/tmt/utils.py#L6107. What we need is a page in docs on (Jinja2) templates (we don't have one, I'll try to write it today), and then developers like @therazix can simply point to it: "we're using Jinja2 templates as supported by tmt, see *this* page".

Reply:
Currently, only the default filters are supported, and some custom ones, as specified by @happz. However, in the current implementation, users are unable to pass variables from the command line to the template. It seems reasonable to provide this capability. Would it make sense to introduce a command line option for this? For example `-v`/`--variable`?

Reply:
Sounds useful. I never used these templates, if it looks like a good addition to people who use them, I'm in. Although I would recommend leaving that for a follow-up patch, if possible.

Reply:
I mentioned the possibility of using default Jinja template filters in d014a34480bd2e60bfca2197a4f0efa17e93f322 and also provided the directory where default templates are stored.

Regarding the option to pass variables to the template, we can address that in a future patch if needed.